src/main/java/io/numaproj/numaflow/info/Language.java

```diff
@@ -0,0 +1,24 @@
+package io.numaproj.numaflow.info;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Please exercise cautions when updating the values below because the exact same values are defined in other Numaflow SDKs
+ * to form a contract between server and clients.
+ */
+public enum Language {
```

Comment:
Any reason why we didn't go with proto? Since it's common among all the sdks we could have kept a common proto file

Reply:
Good point Yashash. I think we should go with proto, that way we can catch any data format issue during development instead of runtime. @whynowy @vigith thoughts?

Reply:
+1, we need to standardize our proto location before we do that. today we manually sink it between the repos.

Reply:
To be addressed by https://github.com/numaproj/numaflow/issues/693